### PR TITLE
[MTDSA-8360] update dependencies

### PR DESCRIPTION
**Pull request by james-work-account**

### CODE REVIEW

File: app/config/DocumentationController.scala [hunk 1]
```diff
@@ -18,17 +18,16 @@ package config
...
 import uk.gov.hmrc.play.bootstrap.backend.controller.BackendController
 
+import javax.inject.{Inject, Singleton}
+
 @Singleton
 class DocumentationController @Inject()(apiDefinition: ApiDefinitionFactory,
                                         cc: ControllerComponents,
-                                        assets: Assets,
-                                        errorHandler: HttpErrorHandler)
+                                        assets: Assets)
```
Comment (Mbrookesy) [inline app/config/DocumentationController.scala:30]: Why was the errorHandler variable was removed?

File: project/SilencerSettings.scala [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    libraryDependencies ++= Seq(
+      compilerPlugin("com.github.ghik" % "silencer-plugin" % silencerVersion cross CrossVersion.full),
+      "com.github.ghik" % "silencer-lib" % silencerVersion % Provided cross CrossVersion.full
+    ),
+    // silence all warnings on autogenerated files
+    scalacOptions += "-P:silencer:pathFilters=target/.*",
+    // Make sure you only exclude warnings for the project directories, i.e. make builds reproducible
+    scalacOptions += s"-P:silencer:sourceRoots=${baseDirectory.value.getCanonicalPath}"
+  )
+}
```
Comment (Mbrookesy) [inline project/SilencerSettings.scala:35]: So what's this file for?

I'm guessing an alternative to silencer libs?

File: app/config/DocumentationController.scala (hunk 1 above)
Comment (james-work-account) [reply app/config/DocumentationController.scala:30]: It's unused

File: project/SilencerSettings.scala (hunk 2 above)
Comment (james-work-account) [reply project/SilencerSettings.scala:35]: This contains the silencer libs - having everything in one place was the recommended approach on the Confluence page I was following when dealing with silencer upgrade issues. It's all brought into build.sbt in one go.

**Pull request merged**